koswat/dike_reinforcements/reinforcement_profile/reinforcement_profile.py

```diff
@@ -21,3 +21,6 @@ class ReinforcementProfile(ReinforcementProfileProtocol, KoswatProfileBase):
     @property
     def new_ground_level_surface(self) -> float:
         return self.profile_width - self.old_profile.profile_width
+
+    def __str__(self) -> str:
+        return self.output_name.replace(" ", "_")
```

Comment:
This should not be needed, you are changing the behavior of the `__str__` method, if you want underscores then it should be addressed only in the export.